cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -308,6 +309,86 @@ public static <C extends Connectable<C>> void setCgmesTapChangerType(C eq, Strin
         }
     }
 
+    // tap changer is exported as it is modelled in IIDM, always at end 1
+    public static void addTapChangerExtensionsNecessaryForExporting(TwoWindingsTransformer twt) {
```

Comment:
naming suggestion: use `addUpdateCgmesTapChangerExtension` instead of `addTapChangerExtensionsNecessaryForExporting`

Reply:
Done